practice/react-practice-one/src/components/Cards/ProductCard.jsx

```diff
@@ -0,0 +1,94 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './productCard.css';
+import '../Text/text.css';
+
+/* Import components */
+import AddIcon from '../Icons/AddIcon';
+import DeleteIcon from '../Icons/DeleteIcon';
+import Text from '../Text/Text';
+import Button from '../Buttons/Button';
+import EditIcon from '../Icons/EditIcon';
+
+/* Import constants */
+import { CARD_TYPES } from '../../constants';
+
+const ProductCard = ({
+  type,
+  imageSrc,
+  title,
+  description = {},
+  onEditClick,
+  onAddClick,
+  onDeleteClick,
+  text,
+}) => {
+  return (
+    <div className={`${type}`}>
+      {type === CARD_TYPES.ADD && (
+        <a
+          href="javascript:void(0)"
+          className="card card-add-product"
+          onClick={onAddClick}
+        >
+          <AddIcon />
+          <Text variant="secondary" className="text-title-sm">
+            {text}
+          </Text>
+        </a>
+      )}
+      {type === CARD_TYPES.PRODUCT && (
+        <div className="card card-product">
```

Comment:
@JennyDoJD should we need update class name as well?

Reply:
I fixed it